jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/utils/WebDataBinderFactory.java

```diff
@@ -1,23 +1,26 @@
 package uk.gov.homeoffice.digital.sas.jparest.utils;
 
 import com.fasterxml.jackson.databind.util.StdDateFormat;
+import java.time.Instant;
 import java.util.Date;
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import org.springframework.beans.propertyeditors.CustomDateEditor;
 import org.springframework.web.bind.WebDataBinder;
+import uk.gov.homeoffice.digital.sas.jparest.editor.CustomInstantEditor;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class WebDataBinderFactory {
 
   @Getter
-  private static WebDataBinder webDataBinder = initWebDataBinder();
+  private static final WebDataBinder webDataBinder = initWebDataBinder();
 
   private static WebDataBinder initWebDataBinder() {
     var webDataBinder = new WebDataBinder(null);
     var dateFormat2 = new StdDateFormat();
     webDataBinder.registerCustomEditor(Date.class, new CustomDateEditor(dateFormat2, true));
+    webDataBinder.registerCustomEditor(Instant.class, new CustomInstantEditor());
```

Comment:
I've just had a quick look at instant because of the need for the custom editor and found that the instant removes timezone information. Should we be using ZonedDateTime

Reply:
See my response to your following comment